src/lib/gcp_apis.py

```diff
@@ -36,6 +36,10 @@ async def get_all_disabled_apis(context: MetricsContext, token: str, project_id:
                 disabled_services_json = await response.json()
                 disabled_services = disabled_services_json.get("services", [])
                 disabled_apis.update({disable_service.get("config", {}).get("name", "") for disable_service in disabled_services})
+                context.log(f"All disabled APIs {disabled_apis}")
```

Comment:
I'm not sure if we want to log all disabled apis, if I remember correctly in the response there are many more apis that we don't support